fem/bilininteg_vecdiffusion.cpp

```diff
@@ -166,6 +216,9 @@ void VectorDiffusionIntegrator::AssemblePA(const FiniteElementSpace &fes)
       auto W = w.Read();
       auto J = Reshape(j.Read(), NQ, SDIM, DIM, ne);
       auto D = Reshape(d.Write(), NQ, SDIM, ne);
+
+      auto C = Reshape(coeff.HostWrite(), NQ, ne);
```

Comment:
Shouldn't this be `coeff.Read()` and doesn't it need to take into account the special case where `coeff` has only one entry?

Reply:
@pazner Yes. Thanks for catching that. This is fixed by a250079153d197c1ee6cfb1e3b7540e7dd839668.